lib/msal-common/src/request/BrokerParameters.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Broker parameters
+ */
+export type BrokerParameters = {
+    // Embedded client id
+    embeddedClientId: string;
+    // Broker client id
+    brokerClientId: string;
```

Comment:
Do we need both clientIds defined here since clientId is already defined at the top level?

Reply:
Correction: I understand this is a breaking change for hosts, so may be we can add a notice to prefer first order over extraQueryParams.

Reply:
Simplified this to a single `embeddedClientId` first order parameter. Other params are picked up from config and request payload.